### PR TITLE
Use `Require all granted` directives with Apache 2.4

### PR DESCRIPTION
**Pull request by iliajie**
Use `Require all granted` directives with Apache 2.4

### CODE REVIEW

File: feature-webalizer.pl [hunk 1]
```diff
@@ -641,7 +641,7 @@ sub create_webalizer_htaccess
 &print_tempfile(HTACCESS, "AuthUserFile $passwd_file\n");
 &print_tempfile(HTACCESS, "require valid-user\n");
 &print_tempfile(HTACCESS, "<Files .stats-htpasswd>\n");
-&print_tempfile(HTACCESS, "deny from all\n");
+&print_tempfile(HTACCESS, "Require all denied\n");
```
Comment (jcameron) [inline feature-webalizer.pl:644]: What about older Apache versions?

Comment (iliajie) [reply feature-webalizer.pl:644]: I suggest we ignore it for Virtualmin. Neither distros have it (Apache 2.2), not even CentOS 7. With newer versions of Virtualmin we should focus on systems that are not EOL.

Comment (jcameron) [reply feature-webalizer.pl:644]: Couldn't we just add a simple test for the Apache version here though?

Comment (iliajie) [reply feature-webalizer.pl:644]: I was about to do this (before the commit) but then realized that we're simply going to make Virtualmin code harder for no reason.

Do you really want to add that kind of a test?

Comment (jcameron) [reply feature-webalizer.pl:644]: You could do it with code like : 

```
&require_apache();
if ($apache::httpd_modules{'core'} < 2.4) {
    &print_tempfile(HTACCESS, "deny from all\n");
} else {
    &print_tempfile(HTACCESS, "Require all denied\n");
}
```

Comment (iliajie) [reply feature-webalizer.pl:644]: Yes, of course. I will fix it if you believe it's still needed.

Comment (jcameron) [reply feature-webalizer.pl:644]: Yes, please do.

File: php-lib.pl [hunk 2]
```diff
@@ -1320,7 +1320,7 @@ sub save_domain_php_directory
 			local @lines = (
 				"    <Directory $dir>",
 				"        Options +IncludesNOEXEC +SymLinksifOwnerMatch +ExecCGI",
-				"        allow from all",
+				"        Require all granted",
```
Comment (jcameron) [inline php-lib.pl:1323]: Also make this conditional on the Apache version

Comment (iliajie) [reply php-lib.pl]: Okay, that is done.

Comment (iliajie) [reply php-lib.pl]: https://github.com/virtualmin/virtualmin-gpl/pull/424/commits/de8365054fb37206a16482cb5c4feb6874079af3

**Pull request merged**